### ISSUE TITLE
Main Thread Checker: UI API called on background thread

### ISSUE THREAD

**Issue by underbjerg** [adform/adform-tracking-ios-sdk#12]
Hi,

When running our app in Xcode 9 beta with AdformTracking 1.1.3, the XCode Main Thread Checker is throwing runtime errors, which seem to be caused by AFWebviewOperation incorrectly calling UIWebView loadRequest on background thread.

### Logs

```
Main Thread Checker: UI API called on a background thread: -[UIWebView loadRequest:]
PID: 1933, TID: 9955590, Thread name: (none), Queue name: NSOperationQueue 0x60800022e5a0 (QOS: UNSPECIFIED), QoS: 0
Backtrace:
4   Tilbud-beta                         0x000000010d057553 -[AFWebviewOperation executeOperation] + 190
5   Foundation                          0x000000010fbcccff __NSOQSchedule_f + 369
6   libdispatch.dylib                   0x00000001161462b5 _dispatch_client_callout + 8
7   libdispatch.dylib                   0x000000011614c232 _dispatch_continuation_pop + 500
8   libdispatch.dylib                   0x000000011614aa93 _dispatch_async_redirect_invoke + 760
9   libdispatch.dylib                   0x0000000116150a72 _dispatch_root_queue_drain + 568
10  libdispatch.dylib                   0x00000001161507dc _dispatch_worker_thread3 + 119
11  libsystem_pthread.dylib             0x00000001165fc5a2 _pthread_wqthread + 1299
12  libsystem_pthread.dylib             0x00000001165fc07d start_wqthread + 13'
```

**Comment by andriusmisiunas** [comment]
Hello,

Thank you for notifying us. There haven't been similar issues on older Xcode versions. We will investigate and let you know by the end of this week.

Best regards,
Andrius Misiunas

**Comment by underbjerg** [comment]
Great, thanks a lot.

**Comment by vladasdrejeris** [comment]
Hello,

We have solved this issue in 1.1.4 SDK version, that was just released.

**Comment by underbjerg** [comment]
Awesome, it works. Thanks for the quick turnaround!
